tapdb/sqlc/queries/mssmt.sql

```diff
@@ -51,6 +51,9 @@ WITH subtree_cte (
 -- name: DeleteNode :execrows
 DELETE FROM mssmt_nodes WHERE hash_key = $1 AND namespace = $2; 
 
+-- name: DeleteRoot :execrows
+DELETE FROM mssmt_roots WHERE namespace = $1;
```

Comment:
Still getting through the diff, but perhaps we should add another variant of `DeleteNode` that also deletes just based on namespace. Or we can modify things to ignore the namespace with a `sqlc.narg` if it isn't set.

Reply:
Made a separate call with a more distinct name.